lte/gateway/c/oai/oai_mme/oai_mme.c

```diff
@@ -136,9 +136,9 @@ int main(int argc, char* argv[]) {
 #endif
   CHECK_INIT_RETURN(s1ap_mme_init(&mme_config));
 
-  if (mme_config.enable_converged_core) {
-    CHECK_INIT_RETURN(ngap_amf_init(&amf_config));
-  }
+  //  if (mme_config.enable_converged_core) {
+  CHECK_INIT_RETURN(ngap_amf_init(&amf_config));
+  //  }
```

Comment:
why are these commented out?

Reply:
This check   "if (mme_config.enable_converged_core) " is failing in Dstester,
thus ngap ports were not coming up. However, this check passes in local VM,
there seems some config related issue in Dstester, thus disabled it.